docs/api/api/api.txt

```diff
@@ -151,6 +151,8 @@ XmlResult: status
 
 Parameters:
   cmd: change_password
+  cmd: lock             # will lock the user and his home projects
+  cmd: delete           # will mark the user as deleted and remove her home projects
```

Comment:
yes, this is the current implementation, I didn't touch that here. This command is just another way to set the state of the user while also changing his home directories.